src/core/content/connector.ts

```diff
@@ -307,7 +307,9 @@ export default class BaseConnector {
 	 * @param trackArtUrl - Track art URL
 	 * @returns Check result
 	 */
-	public isTrackArtDefault: (trackArtUrl?: string) => boolean = () => false;
+	public isTrackArtDefault: (
```

Comment:
Do we really want to do this? Feels like this function should always return bool